src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -150,6 +213,22 @@ void IExtensionConfigProvider.Initialize(ExtensionConfigContext context)
             this.taskHubWorker.AddOrchestrationDispatcherMiddleware(this.OrchestrationMiddleware);
         }
 
+        private void InitializeForFunctionsV1(ExtensionConfigContext context)
+        {
+#if !NETSTANDARD2_0
+            context.ApplyConfig(this.Options, "DurableTask");
+
+            ILogger logger = context.Config.LoggerFactory.CreateLogger(LoggerCategoryName);
+
+            this.TraceHelper = new EndToEndTraceHelper(logger, this.Options.Tracing.TraceReplayEvents);
+            this.HttpApiHandler = new HttpApiHandler(this, logger);
+            this.connectionStringResolver = new WebJobsConnectionStringProvider();
+            this.orchestrationServiceFactory = new OrchestrationServiceFactory(new OptionsWrapper<DurableTaskOptions>(this.Options), this.connectionStringResolver);
+            this.LifeCycleNotificationHelper = this.CreateLifeCycleNotificationHelper();
+            this.nameResolver = context.Config.NameResolver;

```

Comment:
Is this change based on the latest, or is it based on the time we forked v2? I recall there was an `ArgumentNullException` caused by the fact that `nameResolver` was initialized after `CreateLifeCycleNotificationHelper()`. The fix was to reorder things so that `nameResolver` was initialized before calling `CreateLifeCycleNotificationHelper()`.

Reply:
Hmm, good call. I'll take a look.